accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,402 @@
+* **ClientCredentialsSecurityTokenManager**
+    - `CreateSecurityTokenAuthenticator(SecurityTokenRequirement, SecurityTokenResolver)`
+    - `CreateSecurityTokenProvider(SecurityTokenRequirement)`
+    - `CreateSecurityTokenSerializer(SecurityTokenVersion)`
+* **CommunicationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **CommunicationObjectAbortedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **CommunicationObjectFaultedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **DuplexChannelFactory**
+    - `.ctor(InstanceContext, Binding, EndpointAddress)`
+    - `.ctor(InstanceContext, Binding, String)`
+    - `.ctor(InstanceContext, String)`
+    - `.ctor(InstanceContext, String, EndpointAddress)`
+* **DuplexClientBase**
+    - `.ctor(InstanceContext)`
+    - `.ctor(InstanceContext, String)`
+    - `.ctor(InstanceContext, String, EndpointAddress)`
+    - `.ctor(InstanceContext, String, String)`
+* **EndpointAddress**
+    - `ApplyTo(Message)`
+* **EndpointIdentity**
+    - `Equals(Object)`
+    - `GetHashCode()`
+* **EndpointNotFoundException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **FaultException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **InvalidMessageContractException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **MessageHeaderException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **MessageSecurityOverTcp**
+    - `get_ClientCredentialType()`
+    - `set_ClientCredentialType(MessageCredentialType)`
+* **NetHttpBinding**
+    - `CreateBindingElements()`
+* **NetHttpsBinding**
+    - `.ctor()`
+    - `.ctor(BasicHttpsSecurityMode)`
+    - `CreateBindingElements()`
+* **NetTcpBinding**
+    - `.ctor(String)`
+    - `CreateBindingElements()`
+* **ProtocolException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **QuotaExceededException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **ServerTooBusyException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **ServiceActivationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+
+### System.ServiceModel.Channels
+
+* **AddressHeader**
+    - `Equals(Object)`
+    - `GetHashCode()`
+* **MessageHeaders**
+    - `set_To(Uri)`
+* **SecurityBindingElement**
+    - `CreateSecureConversationBindingElement(SecurityBindingElement)`
+* **TransportSecurityBindingElement**
+    - `GetProperty<T>(BindingContext)`
+* **WebSocketTransportSettings**
+    - `get_DisablePayloadMasking()`
+    - `set_DisablePayloadMasking(Boolean)`
+
+### System.ServiceModel.Security
+
+* **MessageSecurityException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **SecurityAccessDeniedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **SecurityNegotiationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **X509ServiceCertificateAuthentication**
+    - `set_CertificateValidationMode(X509CertificateValidationMode)`
+
```

Comment:
Per @mconnew's comment, this suggestion will remove the `ServiceModel` types.

```suggestion
```

Reply:
Do we know what caused them to show up here in the first place?